src/modules/select/wallets/huobiwallet.ts

```diff
@@ -0,0 +1,74 @@
+import { mobileWalletInstallMessage } from '../content'
+import {
+  WalletModule,
+  Helpers,
+  InjectedWithBalanceOptions
+} from '../../../interfaces'
+
+import huobiwalletIcon from '../wallet-icons/icon-huobiwallet'
+
+function huobiwallet(options: InjectedWithBalanceOptions): WalletModule {
+  const { preferred, label, svg, rpcUrl } = options
+
+  return {
+    name: label || 'Huobi Wallet',
+    svg: svg || huobiwalletIcon,
+    wallet: async (helpers: Helpers) => {
+      const { getProviderName, getAddress, getNetwork, getBalance } = helpers
+      const huobiwalletProvider =
+        (window as any).ethereum ||
+        ((window as any).web3 && (window as any).web3.currentProvider)
+
+      const isHuobiWallet = getProviderName(huobiwalletProvider) === 'huobiwallet'
```

Comment:
```suggestion
      const isHuobiWallet = 
        getProviderName(huobiwalletProvider) === 'huobiwallet'
```

Formatting change to satisfy linting rules for line length.